sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -0,0 +1,293 @@
+## API Report File for "@azure/monitor-query"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { PipelineOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export type AggregationType = "None" | "Average" | "Count" | "Minimum" | "Maximum" | "Total";
+
+// @public
+export interface BatchQuery {
+    azureResourceIds?: string[];
+    includeQueryStatistics?: boolean;
+    qualifiedNames?: string[];
+    query: string;
+    serverTimeoutInSeconds?: number;
+    timespan?: string;
+    workspace: string;
+    workspaceIds?: string[];
+    workspaces?: string[];
+}
+
+// @public
+export type ColumnDataType = string;
+
+// @public
+export const CommonDurations: {
+    readonly last7Days: "P7D";
+    readonly last3Days: "P3D";
+    readonly last2Days: "P2D";
+    readonly lastDay: "P1D";
+    readonly lastHour: "PT1H";
+    readonly last4Hours: "PT4H";
+    readonly last24Hours: "P1D";
+    readonly last48Hours: "P2D";
+    readonly last30Minutes: "PT30M";
+    readonly last5Minutes: "PT5M";
+};
+
+// @public
+export interface ErrorDetail {
+    additionalProperties?: any;
+    code: string;
+    message: string;
+    resources?: string[];
+    target?: string;
+    value?: string;
+}
+
+// @public
+export interface ErrorInfo {
+    additionalProperties?: any;
+    code: string;
+    details?: ErrorDetail[];
+    innererror?: ErrorInfo;
+    message: string;
+}
+
+// @public
+export interface GetMetricDefinitionsOptions extends OperationOptions {
+    metricNamespace?: string;
+}
+
+// @public
+export interface GetMetricDefinitionsResponse {
+    definitions: MetricDefinition[];
+}
+
+// @public
+export interface GetMetricNamespacesOptions {
+    startTime?: string;
+}
+
+// @public
+export interface GetMetricNamespacesResponse {
+    namespaces: MetricNamespace[];
+}
+
+// @public
+export interface LocalizableString {
+    localizedValue?: string;
+    value: string;
+}
+
+// @public
+export class LogsClient {
+    constructor(tokenCredential: TokenCredential, options?: LogsClientOptions);
+    // (undocumented)
```

Comment:
ref docs

Reply:
Was leaving these unfilled in on purpose so I can make sure I don't forget to fill them in with actual content. 

Will be happening in the next few PRs.

Reply:
(also captured in my favorite issue: https://github.com/Azure/azure-sdk-for-js/issues/15296